### DISCUSSION TITLE
how to expose restier odata to swagger

### DISCUSSION THREAD

**Discussion by DennyFiguerres** [OData/RESTier#733] [General]
i think there are some ways to get from a to be but which way will work better ?

one is to get restier to use the new v8 library so that another package will work....
https://github.com/OData/RESTier/issues/720

another is to pull the metadata and run it thru a converter in javascript and feed that file to swagger .....

https://oasis-tcs.github.io/odata-openapi/lib/

or is there a way to get restier to output some data that swagger will pickup directly ?

which way to go ???

**Comment by robertmclaws** [comment]
I think we got this resolved with v1.1, so I'm going to go ahead and close it! LMK if you think it should be reopened.
